tensorflow/compiler/xla/service/gpu/gpu_fusible.h

```diff
@@ -69,6 +76,16 @@ bool IsInputFusibleScatter(const HloInstruction& instr);
 bool ShapesCompatibleForMultiOutputFusion(const HloInstruction& instr1,
                                           const HloInstruction& instr2);
 
+// Whether the instructions are compatible for producer-consumer fusion
+// i.e. whether the producer and consumer are loop/input fusible and
+// they are not library calls.
+bool IsProducerConsumerFusible(const HloInstruction& producer,
+                                   const HloInstruction& consumer);
+
+// Whether the instructions are producer-consumer fusible and
+// multi-output fusible.
```

Comment:
I think we should explain what multi-output fusible means briefly here. What about the following:
Whether the instructions are producer-consumer fusible with multiple outputs. That is, the root tuple of the multi-output fusion will contain the results of both, the producer and consumer.